src/Servant/API/BrowserHeader.hs

```diff
@@ -34,7 +34,11 @@ instance HasLink sub => HasLink (BrowserHeader sym a :> sub) where
   toLink _ = toLink (Proxy :: Proxy (Header sym a :> sub))
 #endif
 
-instance (KnownSymbol sym, FromHttpApiData a, HasServer sublayout context)
+instance ( KnownSymbol sym
+         , FromHttpApiData a
+         , HasServer sublayout context
+         , HasContextEntry (context .++ DefaultErrorFormatters) ErrorFormatters
```

Comment:
Could you quickly explain what this is and why it was added?

Thanks!

Reply:
Certainly.  The only reason is the compilation error.  I do not fully understand this.
```
% cabal build         
....
src/Servant/API/BrowserHeader.hs:42:17: error:
    • Could not deduce (HasContextEntry
                          (context .++ DefaultErrorFormatters) ErrorFormatters)
        arising from a use of ‘route’
      from the context: (KnownSymbol sym, FromHttpApiData a,
                         HasServer sublayout context)
        bound by the instance declaration
        at src/Servant/API/BrowserHeader.hs:(37,10)-(38,61)
    • In the expression:
        route (Proxy :: Proxy (Header sym a :> sublayout))
      In an equation for ‘route’:
          route Proxy = route (Proxy :: Proxy (Header sym a :> sublayout))
      In the instance declaration for
        ‘HasServer (BrowserHeader sym a :> sublayout) context’
   |
42 |   route Proxy = route (Proxy :: Proxy (Header sym a :> sublayout))
   |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ```

Reply:
Thanks!